client/hooks/useForm.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
Better explain why it was disabled, as it isn't obvious at first sight. If it's a technical debt, please add a `TODO` comment.